onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.h

```diff
@@ -262,6 +263,9 @@ class TensorrtExecutionProvider : public IExecutionProvider {
   cudnnHandle_t external_cudnn_handle_ = nullptr;
   cublasHandle_t external_cublas_handle_ = nullptr;
 
+  // Call cudaStreamSynchronize() before TRT enqueueV2()/enqueueV3()
```

Comment:
comment says before. but should be after?

Reply:
fixed